### ISSUE TITLE
Moderator identified by viewing member list.

### ISSUE THREAD

**Issue by drickerusa** [RocketChat/Rocket.Chat#4959]
Your Rocket.Chat version: 0.45.0
My user community has been asking me how to view who the moderator is in that current channel. Unless I'm missing something here I can identity the moderator if they share a message where the tag identifies them. Can an enhancement be made when you go into a channel and view the member list to put a tag on the moderator and even the owner.

**Comment by baynezy** [comment]
Yes please. This is hugely annoying, I should be able to identify each channel member's role without needing them to send a message to the channel.

**Comment by cpitman** [comment]
I get this question from users frequently for larger channels, it would be a big help!

**Comment by hexcsl** [comment]
The problem exists and it is not very convenient.
Users ask who the owner is and who is the moderator in the channel. This can only be known by the history of the message or by looking at the information for each user.

P.S. I'm using version Rocket.Chat 0.62.2 and 0.60.3.

**Comment by geekgonecrazy** [comment]
#14319 related
